rmf_rviz_plugin/src/ParseActionPlan.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef SRC_RMF_RVIZ_PLUGIN__PARSEACTIONPLAN_HPP
+#define SRC_RMF_RVIZ_PLUGIN__PARSEACTIONPLAN_HPP
+
+#include <rmf_task_msgs/msg/delivery.hpp>
+#include <rmf_task_msgs/msg/loop.hpp>
+#include <rmf_utils/optional.hpp>
+#include <QTimeEdit>
+
+namespace rmf_rviz_plugin {
+
+using Delivery = rmf_task_msgs::msg::Delivery;
```

Comment:
consider adding these `using` lines as well as the function into a separate namespace, as all the source code for these plugin is under the `rmf_rviz_plugin` namespace, conflicts might occur, for example, `RmfPanel.hpp` also has this line defined under the same namespace.

Reply:
i would like to make sure i'm doing this correctly as i'm not too familiar, but is this what you might like to do? 
https://github.com/cnboonhan94/rmf_demos/commit/26f84dddb9ed6a0a86138115ed93ddf620a29808

Reply:
I'm wondering if its a better idea to move https://github.com/cnboonhan94/rmf_demos/blob/26f84dddb9ed6a0a86138115ed93ddf620a29808/rmf_rviz_plugin/src/ParseActionPlan.hpp#L12-L16 into the definition of `parse_yaml_config()` as implementation details, so the header `ParseActionPlan.hpp` only has a single function declaration? 

this way we can actually do away with the namespace `utils` too. Sorry I probably should have made it clearer in our slack conversation

Reply:
oh right! woops. its not an issue to refactor this. however, in the spirit of this, this should probably need to be moved as well https://github.com/cnboonhan94/rmf_demos/blob/new_interface/rmf_rviz_plugin/src/RmfPanel.hpp#L67

Reply:
:+1: